### ISSUE TITLE
kibana service is failing in ELK 7.3. 

### ISSUE THREAD

**Issue by Venkat639** [spujadas/elk-docker#295]
deploying latest(730) ELK in kubernetes cluster. Elasticsearch and logstash are coming up but kibana is failing with error "start-stop-daemon: warning: failed to kill 23: No such process"

**Comment by tedder** [comment]
Where do you see that error? I'm using 740 and kibana fails to start, but I can't find any errors. If I exec in and run `service kibana start` it'll start up. This is what I see n the elk-docker log:
```
waiting for Elasticsearch to be up (20/90)
Waiting for Elasticsearch cluster to respond (1/30)
logstash started.
 * Starting Kibana5
   ...done.
```

When I exec in the kibana log is empty. Of course, doing `start` causes it to run and the logs to fill.

**Comment by tedder** [comment]
Figured this out. It was failing during `start` too. Nothing in the logs indicating a crash (even the OS logs in the container or on the server). I finally figured it out- I needed to use the NODE_OPTIONS environment var:
```
NODE_OPTIONS="--max_old_space_size=4096"
```

While running a hacked init.d service so `--verbose` was used to start Kibana, I could see memory usage mostly around 540mb but it would be in the 800mb range, probably during optimize. I'm not installing any Kibana plugins so I wonder if the default size needs to be increased from 512, @spujadas.

[The memory usage in Kibana is a known and open issue](https://github.com/elastic/kibana/issues/19678).

see also: #104 #105 #17 #176
